Kernel/Net/IPv4Socket.cpp

```diff
@@ -722,17 +722,15 @@ ErrorOr<void> IPv4Socket::ioctl(OpenFileDescription&, unsigned request, Userspac
             return ENODEV;
         }
 
-        char namebuf[IFNAMSIZ + 1];
-        memcpy(namebuf, ifr.ifr_name, IFNAMSIZ);
-        namebuf[sizeof(namebuf) - 1] = '\0';
-
+        auto namebuf = Array<u8, IFNAMSIZ + 1> {};
+        memcpy(namebuf.data(), ifr.ifr_name, IFNAMSIZ);
```

Comment:
This wrote the entire storage on the LHS, no need for the `{}` (zero-filling the whole array just for the last byte is a bit silly).